src/core/constructs/output_manager.py

```diff
@@ -287,7 +285,41 @@ def _print_component_resource_differences(
                 self._console.print(
                     f"        [bold yellow]Update:[/bold yellow][bold blue] {resource_diff.new_resource.name} ({resource_diff.new_resource.ruuid})[/bold blue]"
                 )
-
+                for key in resource_diff.new_resource.dict().keys():
+                    if resource_diff.previous_resource.dict().get(
+                        key
+                    ) != resource_diff.new_resource.dict().get(key):
+                        if type(resource_diff.previous_resource.dict().get(key)) == frozenset:
+                            event = resource_diff.previous_resource.dict().get(key)
```

Comment:
can convert to normal list with `list(resource_diff.previous_resource.dict().get(key))`